metagraph/src/graph/representation/succinct/build_checkpoint.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <cstdint>
+#include <filesystem>
+#include <fstream>
+
+namespace mtg {
+namespace graph {
+namespace boss {
+
+/**
+ * Stores checkpointing and phase information for resuming disk-based building of succinct
```

Comment:
The code shows that there are >4 checkpoints, but the help message in config only mentions two possible values for phase. It would be useful to describe here how checkpoints and phases are related.